x-pack/plugins/security_solution/public/resolver/models/resolver_tree.ts

```diff
@@ -8,10 +8,28 @@ import {
   ResolverTree,
   ResolverNodeStats,
   ResolverLifecycleNode,
-  ResolverChildNode,
   SafeResolverEvent,
+  NewResolverTree,
+  ResolverNode,
+  EventStats,
 } from '../../../common/endpoint/types';
-import * as eventModel from '../../../common/endpoint/models/event';
+import * as nodeModel from '../../../common/endpoint/models/node';
+
+/**
+ * This returns a map of nodeIds to the associated stats provided by the datasource.
+ */
+
```

Comment:
I don't think tsdoc will acknowledge the above doc comment if there is this extra new line.